src/main/java/org/reaktivity/nukleus/sse/internal/SseConfiguration.java

```diff
@@ -18,19 +18,25 @@
 import org.agrona.DirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
 import org.reaktivity.nukleus.Configuration;
+import org.reaktivity.nukleus.sse.internal.types.StringFW;
 
 public class SseConfiguration extends Configuration
 {
+    public static final String EVENT_TYPE_NAME = "nukleus.sse.event.type";
+
     public static final BooleanPropertyDef SSE_INITIAL_COMMENT_ENABLED;
 
     private static final DirectBuffer INITIAL_COMMENT_DEFAULT = new UnsafeBuffer(new byte[0]);
 
     private static final ConfigurationDef SSE_CONFIG;
 
+    static final PropertyDef<String> EVENT_TYPE;
```

Comment:
Please rename this to `CHALLENGE_EVENT_TYPE` and update related constants, method names etc.